### DISCUSSION TITLE
Debugging Tiled Extensions

### DISCUSSION THREAD

**Discussion by mcarpenter622** [mapeditor/tiled-extensions#24] [Team Posts]
Ok this is probably a dumb question, but does anyone know how to see debug logs in Tiled js extensions? I know for the godot exporter there are some log methods but for the life of me I cannot figure out where they log to? It doesn't appear to log to the console in the Tiled editor.

**Comment by bjorn** [comment]
Apparently [this extension is using `console.log`](https://github.com/mapeditor/tiled-to-godot-export/blob/bf564bf02b58eeb46dd8a92f066c33c85a65f9bc/utils.js#L4-L11). This used to get routed to the Console view, as part of routing all kinds of debug output there. Since Tiled 1.8 only actual script errors automatically go to the Console (see also issue https://github.com/mapeditor/tiled/issues/3010).

So, the fix is to replace `console.log` with `tiled.log`, which has always been available specifically for writing to the Console view. Just keep in mind that unfortunately I couldn't make `tiled.log` accept multiple parameters.

Alternatively, if you launch Tiled from a terminal then the `console.log` output should be visible there.
